en_US/deploy/cluster/db.md

```diff
@@ -9,19 +9,19 @@ EMQX uses an embedded [Mria](https://github.com/emqx/mria) database to store the
 - etc.
 
 Mria tables are replicated across all EMQX nodes.
-It also helps with the fault-tolerance: the data is safe, as long as at least one node in the cluster is alive.
+It also helps with the fault-tolerance: the data is safe as long as at least one node in the cluster is alive.
 
-If the size of the EMQX cluster is below 5 nodes, typically no database scalability tuning is required.
+If the size of the EMQX cluster is below 5 nodes, typically, no database scalability tuning is required.
```

Comment:
I think these commas are not necessary, English is very lax about commas

Reply:
AI checker (grammarly) always want commas around introductory phrases 😅